src/components/MaterialUI/DataDisplay/modalProps.js

```diff
@@ -7,6 +7,8 @@ class ModalProps extends ComponentProps {
 		actionPanel: "actionPanel",
 		backdropClickCallback: "backdropClickCallback",
 		type: "type",
+		okButtonLabel: "okButtonLabel",
+		cancelButtonLabel: "cancelButtonLabel",
```

Comment:
it was supposed that such buttons will be part of action panel. so, why would you need those props in modalProps?